deploy/servctl_utils/deploy_command_utils.py

```diff
@@ -182,6 +185,29 @@ def _set_elasticsearch_kubernetes_resources():
         run('kubectl apply -f deploy/kubernetes/elasticsearch/kubernetes-elasticsearch-all-in-one.yaml')
 
 
+def deploy_elasticsearch_snapshot_infra(settings):
+    print_separator("elasticsearch snapshot infra")
+
+    if settings["ES_CONFIGURE_SNAPSHOTS"]:
+
+        # create the bucket
+        run("gsutil mb -p seqr-project -c STANDARD -l US-CENTRAL1 gs://%(ES_SNAPSHOTS_BUCKET)s" % settings, errors_to_ignore=["already exists"])
+
+        # check if the serviceaccount exists
+        try:
+            run("gcloud iam service-accounts describe %(ES_SNAPSHOTS_ACCOUNT_NAME)s@seqr-project.iam.gserviceaccount.com" % settings)
+        except RuntimeError:
+            # service account does not exist, create it now
+            run("gcloud iam service-accounts create %(ES_SNAPSHOTS_ACCOUNT_NAME)s --display-name %(ES_SNAPSHOTS_ACCOUNT_NAME)s" % settings)
```

Comment:
our general approach everywhere else in servctl is to just run things with errors_to_ignore, rather than explicitly try-catching if things exist. This is okay to leave as is, but I was wondering if there was a reason you picked a different approach here

Reply:
This was just to avoid running the create command multiple times, as I wasn't sure how idempotent it was. Testing now, it just fails if the acct already exists, so I can switch to more familiar logic.